### PR TITLE
Export update

### PR DESCRIPTION
**Pull request by metincansiper**
Refs #685.

I made changes in factoid and unstable branch of factoid-converters (https://github.com/PathwayCommons/factoid-converters/) based on this document (https://docs.google.com/document/d/1CfnJNpbRFmuc7LrtcD7VcQi-sSHM7XrTtKIR5Uo_ShI/edit). Note that Emek and me briefly went over the document the last week and made some further changes there based on the suggestions of Emek. May need to make a few more after asking him some more questions.

### CODE REVIEW

**Comment by metincansiper** [comment]
Probably these changes will not be testable in unstable right after merge. The compatible converter java code is in unstable branch now. 

I have not been involved in deploying the service to the public url (https://biopax.baderlab.org/convert/v2/). How can we update it? Is there a automated deployment there?

If not we can either:

- Initially deploy from unstable branch and then merge to master when we make sure that it works well.
- Merge directly to master and deploy from there.

File: src/model/element/interaction.js [hunk 1]
```diff
@@ -225,8 +225,8 @@ class Interaction extends Element {
     return _.assign( {}, super.json(), _.pick( this.syncher.get(), _.keys(DEFAULTS) ) );
   }
 
-  toBiopaxTemplate(){
-    return this.association().toBiopaxTemplate();
+  toBiopaxTemplate( getElById ){
```
Comment (metincansiper) [inline src/model/element/interaction.js:228]: There probably is a typo here. Looks like ``getElById`` must be named as ``transform``. I will fix it.

**Comment by maxkfranz** [comment]
> I have not been involved in deploying the service to the public url (https://biopax.baderlab.org/convert/v2/). How can we update it? Is there a automated deployment there?

If no one is using it now, then the existing instance could be used for testing.  In future, there should be a separate instance for testing.

**Comment by jvwong** [comment]
I have the info for the production instance. Since this instance is effectively incompatible with the app, we should just update it.

**Comment by jvwong** [comment]
I configured `unstable.factoid.baderlab.org` to point at a local instance of `factoid-converters`(unstable) which is currently running on loopback 9999. @metin do you want to just merge this PR in?

**Comment by metincansiper** [comment]
>I configured unstable.factoid.baderlab.org to point at a local instance of factoid-converters(unstable) which is currently running on loopback 9999. @metin do you want to just merge this PR in?

@jvwong  Sure I can merge that but what address should this line be pointing (https://github.com/PathwayCommons/factoid/blob/export-update/src/config.js#L50)
before merging it?

**Comment by jvwong** [comment]
> @jvwong Sure I can merge that but what address should this line be pointing (https://github.com/PathwayCommons/factoid/blob/export-update/src/config.js#L50)
> before merging it?

Just leave it as is. I have configured `BIOPAX_CONVERTER_URL` as an environment variable for `https://unstable.factoid.baderlab.org/` that points to a factoid-converter instance running locally on the same VM.

**Comment by metincansiper** [comment]
Okay I merged it.

**Pull request merged**